.gitignore

```diff
@@ -1,6 +1,13 @@
 *.log
 *.log*
-config*.json
+config.json
 config*.json~
+!deploy/config.json
+tmp-config.json
 node_modules
 build/*
+*.tags*
+metarpheus.jar
+deploy/build/*
```

Comment:
some of these could go away I think: `config.json`, `deploy/config.json`, `tmp-config.json` and `metarpheus.jar` (should be downloaded and cached by scriptoni elsewhere)